### PR TITLE
Use EventLoopGroup instead of EventLoop

### PR DESCRIPTION
**Pull request by snripa**
Create `AeronEventLoopGroup`. Must host internally an array of `AeronEventLoop`-s. 

Must impl. `OnDisposable` and provide public function `AeronEventLoop next()`.

### CODE REVIEW

File: reactor-aeron-core/src/main/java/reactor/aeron/AeronEventLoop.java [hunk 1]
```diff
@@ -138,6 +139,34 @@ private static ThreadFactory defaultThreadFactory() {
         .switchIfEmpty(Mono.error(Exceptions::failWithRejected));
   }
 
+  /**
+   * Runnable task for submitting to {@link #commandTasks} queue. For usage see methods {@link
+   * #register(MessagePublication)} and {@link #dispose(MessagePublication)}.
+   */
+  private static class CommandTask implements Runnable {
```
Comment (segabriel) [inline reactor-aeron-core/src/main/java/reactor/aeron/AeronEventLoop.java]: what is it change? why is it here? A private class should be below.

Comment (segabriel) [reply reactor-aeron-core/src/main/java/reactor/aeron/AeronEventLoop.java]: I don't understand moving this and `ThreadFactory` too

File: reactor-aeron-core/src/main/java/reactor/aeron/AeronResources.java [hunk 2]
```diff
@@ -302,7 +302,7 @@ public boolean isDisposed() {
                   category,
                   AeronUtils.format(subscription),
                   eventLoop,
-                  ex);
+                  ex.toString());
```
Comment (segabriel) [inline reactor-aeron-core/src/main/java/reactor/aeron/AeronResources.java]: what a reason use `toString`?

File: reactor-aeron-tools/pom.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.scalecube</groupId>
     <artifactId>scalecube-reactor-aeron</artifactId>
-    <version>0.0.8</version>
+    <version>0.0.9-SNAPSHOT</version>
```
Comment (segabriel) [inline reactor-aeron-tools/pom.xml]: maybe need to rebase on `develop` branch

File: reactor-aeron-core/src/main/java/reactor/aeron/AeronResources.java (hunk 2 above)
Comment (artem-v) [reply reactor-aeron-core/src/main/java/reactor/aeron/AeronResources.java]: No to write stack

**Pull request merged**